src/main/java/myPackage/Deadlines.java

```diff
@@ -0,0 +1,56 @@
+package myPackage;
+
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadlines extends Task{
+    private String deadline;
+    private LocalDate localDate;
+
+    public Deadlines(String description, String deadline) {
+        super(description);
+        try {
+            String[] str = deadline.split("by ");
```

Comment:
Perhaps, the use of a plural noun such as `strings` will be clearer that this is an array of `String`